src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -48,13 +48,24 @@ public static String convertWordToPigLatin(String word) {
 	
 	
 	public static String convertStringToPigLatin(String inputString) {	
-		String[] words = inputString.split(" ");
+        inputString = inputString.trim(); 
+        String[] words = inputString.split("\\W+");
+        String[] delimiters = inputString.split("\\w+");
 		String output = new String();
+        // The following line does not wor´k, why? does it need to work?
+        //output += delimiters[0]; 
```

Comment:
So, should you remove these then? Or? I cannot say if it needs to or not however...